src/main/scala/org/holmesprocessing/totem/services/zipmeta/Dockerfile

```diff
@@ -25,9 +17,9 @@ COPY README.md /service
 COPY ZipParser.py /service
 COPY extra_field_parse.py /service
 COPY zipmeta.py /service
-
+COPY holmeslibrary /service/holmeslibrary
 # add the configuration file (possibly from a storage uri)
 ARG conf=service.conf
 ADD $conf /service/service.conf
 
-CMD ["python2", "/service/zipmeta.py"]
+CMD ["python3", "/service/zipmeta.py"]
```

Comment:
the issue with zipmeta might be the conversion from python2 to python3. can you confirm please?